Dev

jetminded opened the pull request:
Py project

jetminded committed:
1st commit

jetminded committed:
fixed readme

jetminded committed:
Update README.md

jetminded committed:
Update README.md

jetminded committed:
Update README.md

jetminded committed:
Update README.md

jetminded committed:
Update README.md

jetminded committed:
Update README.md

jetminded committed:
Update README.md

jetminded committed:
Update README.md

jetminded committed:
Update README.md

jetminded committed:
Update README.md

jetminded committed:
Update README.md

jetminded committed:
Update README.md

jetminded committed:
Update README.md

jetminded committed:
Update README.md

jetminded committed:
Update README.md

jetminded committed:
Update README.md

jetminded committed:
second-to-last fixes

jetminded committed:
Update README.md

jetminded committed:
Update README.md

jetminded committed:
Update README.md

jetminded committed:
Update README.md

jetminded committed:
Update README.md

jetminded committed:
Update README.md

jetminded committed:
Update README.md

jetminded committed:
Update README.md

jetminded committed:
~final~ commit

lajulienn commented on dice_lib.py:
```diff
@@ -0,0 +1,201 @@
+        plt.show()
+
+
+def geqslant(arr, modif, x):
```
Некоторые названия функций не очень понятны. Лучше подлиннее, но понятно. Аналогично с аргументами: modif...?

lajulienn commented on dice_lib.py:
```diff
@@ -0,0 +1,201 @@
+        self.add(s)
+
+    def add(self, s):
+        if s != '':
```
Что такое s?

lajulienn commented on dice_lib.py:
```diff
@@ -0,0 +1,201 @@
+                x = cpy
+        return x
+
+    def bmb(self, depth):
```
Снова непонятное название

lajulienn commented on main.py:
```diff
@@ -0,0 +1,173 @@
+    return round(prob_sum / sum(arr), accuracy)
+
+
+def character_build():
```
Давай эти функции тоже из main.py вынесем и оставим тут только самое основное для запуска приложения

lajulienn commented on dice_lib.py:
```diff
@@ -0,0 +1,201 @@
+import matplotlib.pyplot as plt
```
Нужно добавить файл с зависимостями:
`pip freeze > requirements.txt`

lajulienn commented on main.py:
```diff
@@ -0,0 +1,173 @@
+        character_build()
+    else:
+        cycle_dice()
+    print('Thanks for using my program! <3')
```
В конце файла по pep8 - перевод строки. Проверь в целом, чтобы весь код соответствовал кодстайлу. В pycharm можно включить автопроверку

lajulienn reviewed (changes requested):
Полезная штука! Осталось исправить пару замечаний

jetminded committed:
commit fixes
- refactored code according to PEP-8
- added extra comments to describe functions
- added requirements.txt
